ice_setup/ice.py

```diff
@@ -567,6 +575,7 @@ def create_repo_file(cls, template_name, repo_url, gpg_url, file_name=None, **kw
             contents = template.format(
                 gpg_url=gpg_url,
                 repo_url=repo_url,
+                gpg_check=1 if use_gpg else 0,
```

Comment:
Good question which raises the generic one: what should go in a first class args and what should go in kw?  I tend to prefer args with defaults if only for the quick documentation of options.